### ISSUE TITLE
Column with larger content is skipping first page and table borders are not displayed for content overflown to the next page

### ISSUE THREAD

**Issue by GeniusGo** [simonbengtsson/jsPDF-AutoTable#256]
Hello, I'm using this beautiful plugin.
 I'm facing 2 issues while using this plugin:
   **1. If the content in the column is larger, then it skips the first page and starts from the next page.**
 
![screenshot--2017-04-25-22-41-11](https://cloud.githubusercontent.com/assets/23277581/25398128/79633a96-2a08-11e7-8c3a-d5d3e4558e91.png)

   **2. If the content in the column is larger, then the remaining content in that column goes to next line, but it does not get any table borders continued along with it.**

![screenshot--2017-04-25-22-41-52](https://cloud.githubusercontent.com/assets/23277581/25398142/88a5161e-2a08-11e7-8790-250e0a75bbf3.png)

I've configured the plugin for my project as:
```
 doc.autoTable(columns, rows, {
      drawHeaderRow: function (row, data) {
        if (data.pageCount == 1) {
          doc.text("List Of Subcontractors", 40, 45);
        }
      },
      drawRow: function (row) {

      },
      theme: 'grid',
      pageBreak: 'avoid',
      showHeader: 'firstPage',
      headerStyles: {
        fillColor: [27, 185, 52],
        textColor: [255],
        halign: 'center'
      },
      bodyStyles: {
        halign: 'left',
        textColor: [53, 64, 82],
        overflow: 'linebreak'
      },
      margin: {top: 60},
    });
    doc.setFontSize(50);
    doc.save('subcontractors.pdf');  

```

**Comment by simonbengtsson** [comment]
Will be fixed in version 3.0.

**Comment by simonbengtsson** [comment]
Duplicate of #370
